### DISCUSSION TITLE
spdlog wrok on with log crate

### DISCUSSION THREAD

**Discussion by youngday** [SpriteOvO/spdlog-rs#8] [Q&A]
we can use log crate and log4rs crate together, but could not use spdlog-rs crate  with log crate together,how to do it ？

**Comment by SpriteOvO** [comment]
Since `log` crate does not support the feature of multiple loggers, `spdlog-rs` has reimplemented most of the stuff.
By calling [`init_log_crate_proxy()`](https://docs.rs/spdlog-rs/0.2.3/spdlog/fn.init_log_crate_proxy.html) to enable the proxy layer for `log` crate, it will forward all messages from `log` crate to `spdlog-rs`'s logger.
